Source/ConstraintViewDSL.swift

```diff
@@ -99,3 +99,84 @@ public struct ConstraintViewDSL: ConstraintAttributesDSL {
     }
     
 }
+
+
+#if os(iOS) || os(tvOS)
+@available(iOS 11.0, *)
+public extension ConstraintViewDSL {
+    
+    func toCenter() { view.snp.makeConstraints { make in make.center.equalToSuperview() } }
+       
+       func toTopCenter(_ inset: CGFloat = 0, toSafeArea: Bool = false) {
```

Comment:
Something is wrong with spacing. Btw why not set CGFloat with default of 0.0 instead of integer 0?